drivers/soundwire/sysfs_local.h

```diff
@@ -0,0 +1,19 @@
+/* SPDX-License-Identifier: GPL-2.0-only */
+/* Copyright(c) 2015-2020 Intel Corporation. */
+
+#ifndef __SDW_SYSFS_LOCAL_H
+#define __SDW_SYSFS_LOCAL_H
+
+/*
+ * SDW sysfs APIs -
+ */
+
+int sdw_slave_sysfs_init(struct sdw_slave *slave);
+
+void sdw_slave_sysfs_exit(struct sdw_slave *slave);
+
+int sdw_slave_sysfs_dpn_init(struct sdw_slave *slave);
+
+void sdw_slave_sysfs_dpn_exit(struct sdw_slave *slave);
```

Comment:
if these functions belong to a single API, which they seem to do, maybe would be better to not insert empty lines between them.

Reply:
fixed